mmv1/products/identityplatform/Config.yaml

```diff
@@ -73,10 +103,41 @@ properties:
     name: 'autodeleteAnonymousUsers'
     description: |
       Whether anonymous users will be auto-deleted after a period of 30 days
+  - !ruby/object:Api::Type::NestedObject
+    name: 'multiTenant'
+    description: |
+      Configuration related to multi-tenant functionality.
+    properties:
+      - !ruby/object:Api::Type::Boolean
+        name: 'allowTenants'
+        description: |
+          Whether this project can have tenants or not.
+      - !ruby/object:Api::Type::String
+        name: 'defaultTenantLocation'
+        description: |
+          The default cloud parent org or folder that the tenant project should be created under.
+          The parent resource name should be in the format of "/", such as "folders/123" or "organizations/456".
+          If the value is not set, the tenant will be created under the same organization or folder as the agent project.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'monitoring'
+    description: |
+      Configuration related to monitoring project activity.
+    default_from_api: true
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'requestLogging'
+        description: |
+          Configuration for logging requests made to this project to Stackdriver Logging.
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'enabled'
+            description: |
+              Whether logging is enabled for this project or not.
   - !ruby/object:Api::Type::NestedObject
     name: 'signIn'
     description: |
       Configuration related to local sign in methods.
+    default_from_api: true
```

Comment:
I had to add to this field as well, it seems the API returns something even when not set

```
=== CONT  TestAccIdentityPlatformConfig_identityPlatformConfigMonitoringExample
    vcr_utils.go:152: Step 1/2 error: After applying this test step, the plan was not empty.
        stdout:


        Terraform used the selected providers to generate the following execution
        plan. Resource actions are indicated with the following symbols:
          ~ update in-place

        Terraform will perform the following actions:

          # google_identity_platform_config.default will be updated in-place
          ~ resource "google_identity_platform_config" "default" {
                id                         = "projects/tf-testi2dmqeauyz/config"
                name                       = "projects/866775349519/config"
                # (3 unchanged attributes hidden)

              - sign_in {
                  - allow_duplicate_emails = false -> null
                  - hash_config            = [
                      - {
                          - algorithm      = "SCRYPT"
                          - memory_cost    = 14
                          - rounds         = 8
                          - salt_separator = "Bw=="
                          - signer_key     = "fxBwaqYujgOuuq3NG/aeDC/9x7c6oar9Hxyrr3GdzRuu+16oBveHuHjKiecAYNJFgHFz+091dPCwgrVnYoef6Q=="
                        },
                    ] -> null
                }

                # (1 unchanged block hidden)
            }

        Plan: 0 to add, 1 to change, 0 to destroy.
```

Reply:
Sorry, it seems like a bug introduced from a new feature last week. I already created a PR to fix it and hopefully the fix will be merged very soon. Thanks for catching it!

Reply:
Thanks @shuyama1 - I saw the fix is in and I have merged main into this PR

Reply:
Thanks!